.travis.yml

```diff
@@ -0,0 +1,10 @@
+install: true
+language: node_js
+node_js:
+  - "iojs"
+  - "7"
+# safelist
+branches:
+  only:
+  - master
+  - dev
```

Comment:
I haven't used continuous integration before, so I don't really know what's typical practice 😅  I just assumed you only wanted the major branches that the team works on to be tests. I'll remove it so that all branches are tested 👍